# origin: Debian, #174554
# cset: n/a
# inclusion: not submitted
# description: comment out leading comment in smp.tex
# revision date: 2004-09-04

diff -urN kernel-source-2.4.26/Documentation/smp.tex kernel-source-2.4.26-1/Documentation/smp.tex
--- kernel-source-2.4.26/Documentation/smp.tex	1998-06-25 07:30:07.000000000 +1000
+++ kernel-source-2.4.26-1/Documentation/smp.tex	2003-01-26 14:11:55.000000000 +1100
@@ -1,8 +1,8 @@
-From: michael@Physik.Uni-Dortmund.DE (Michael Dirkmann)
-
-thanks for your information. Attached is the tex-code of your
-SMP-documentation :
--=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
+% From: michael@Physik.Uni-Dortmund.DE (Michael Dirkmann)
+% 
+% thanks for your information. Attached is the tex-code of your
+% SMP-documentation :
+% -=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
 \documentclass[]{article}
 \parindent0.0cm
 \parskip0.2cm

